### ISSUE TITLE
Click on `irs-bar` changes `options.to`

### ISSUE THREAD

**Issue by gsouquet** [IonDen/ion.rangeSlider#394]
Hi, 

I came across a weird bug, and I can't really get my head around it.
I am using ion.rangeSlider 2.1.4, and jQuery 1.12.4.

This is my HTML:

``` html
<input type="text" id="slider">
```

And the associated JavaScript

``` js
var $slider = $('#slider');

$slider.ionRangeSlider({
  "type": "double",
  "drag_interval": true,
  "min": 0,
  "max": 100,
  "from": 20,
  "to": 40,
  "min_interval": 1,
  "hide_min_max": true
});

var ionSlider = $slider.data("ionRangeSlider");
```

Now just launch the page, and click on the `irs-bar` (the blue bar in the middle). Nothing happens. That is the intented behaviour.

Now try the experiment again and use the following options to instanciate the slider 

``` js
$slider.ionRangeSlider({
  "type": "double",
  "drag_interval": true,
  "min": 0,
  "max": 100000,
  "from": 20000,
  "to": 40000,
  "min_interval": 1000,
  "hide_min_max": true
});
```

And now click on the `irs-bar` and you should see the `to` value change. 
Now the full code I want to use is this: 

``` js
var $slider = $('#slider');

$slider.ionRangeSlider({
  "type": "double",
  "drag_interval": true,
  "min": 0,
  "max": 100000,
  "from": 20000,
  "to": 40000,
  "min_interval": 1000,
  "hide_min_max": true,
  "timeline_click": false
});

var ionSlider = $slider.data("ionRangeSlider");
setInterval(function() {
  ionSlider.update({ max: ionSlider.options.max + 10000 }); 
}, 1000);
```

And I also experience the problem describe above.

I think the problem comes from the `this.coords.p_to_real` in the `calc` function, but I'm not quite sure how all of that works.

Any idea, or suggestion on how to fix that?

**Comment by IonDen** [comment]
Hi, please provide demo here: http://jsfiddle.net/IonDen/726L0aj5/

**Comment by gsouquet** [comment]
Sure, here it is: 
http://jsfiddle.net/726L0aj5/5/

**Comment by IonDen** [comment]
I see it now, yep a bug.

**Comment by gsouquet** [comment]
Is that something that you plan to fix with this version, or are you focusing 100% on the new version?

I'm happy to help but I'm not quite sure where to start.

**Comment by IonDen** [comment]
There will be 1 last bug-fix update (2.1.5) before 3.x release. So, i will try to fix as many bugs, as possible in this version.

**Comment by gsouquet** [comment]
I have noticed that if I comment out `this.force_redraw = true` in the `pointerUp` function the issue goes away.

This could work for me as an interim until a fix is released. I just wanted to know if commenting this out could as some side-effects? 

I'm happy to help you debug further, my understanding of the issue is that there is a rounding issue somewhere. But my lack of knowledge of how the plugin is built kind of blocks me. Let me know if I can help.

Thanks,

**Comment by IonDen** [comment]
Should be fixed in 2.1.5
